### DISCUSSION TITLE
Next.js 새로운 버전

### DISCUSSION THREAD

**Discussion by cyberjam** [LifeIsAlone/public-home#66] [General]
안녕하세요 낭만개발자 @ParkAward 님 
잘 지내시나요
next.js 새로운 버전 13이 나왔더라구요

우리가 스프레드 시트 데이터를 가져올때 사용했던 
getStaticProps, getServerSideProps 이런게 곧 사라지고 fetch 이런걸로 대체될거라네요. 
아직까지는 병행해서 지원한다곤 하더라구요.
반쪽짜리 서버 프레임워크라는 오명을 벗을 수 있을지 Next.js 앞으로 행보가 어떻게 될는지 궁금하군요.

**Comment by sangjuun-park** [comment]
넵 확인하겠습니다. 
반쪽짜리 서버 프레임워크라는 오명이 있군요. 재민님 나중에 NextJS에 대해 여쭤보겠습니다. ㅎㅎ
화이팅 입니다!
